src/main/java/handlers/DefaultHandler.java

```diff
@@ -0,0 +1,19 @@
+package main.java.handlers;
+
+import main.java.server.Response;
+import main.java.server.request.Request;
+
+import static main.java.server.HTTPMessageComponents.HTTPSyntax.*;
+import static main.java.server.HTTPMessageComponents.StatusCodes.NOT_FOUND;
+
+public class DefaultHandler implements Handler {
+    @Override
+    public Response generateResponseForRequest(Request request) {
+        return new Response(VERSION + SP + NOT_FOUND + CRLF);
+    }
+
+    @Override
+    public String getPath() {
+        return null;
```

Comment:
This scares me a bit. I understand why it makes sense - a handler typically maps to a path, but this handler is for paths that aren't recognized. That said, I'm wondering whether this function might end up getting called somewhere that `null` isn't expected/handled. 

No worries if this seems like we want to push off changes until we learn more about the problem space. In the meantime, I'd check out: https://8thlight.com/blog/brian-pratt/2012/12/21/dont-fail-at-failure.html